### ISSUE TITLE
Tabbed Interface: DOMException in browser console

### ISSUE THREAD

**Issue by kombatnt** [wet-boew/wet-boew#8982]
The Tabbed Interface plugin currently produces the following exception in the Developer Tools console of Chrome when running in Chrome 80.0.3987.132

```
Uncaught DOMException: Failed to execute 'replaceChild' on 'Node': The node to be replaced is not a child of this node.
    at q (https://wet-boew.github.io/v4.0-ci/wet-boew/js/wet-boew.min.js:12:11312)
    at p (https://wet-boew.github.io/v4.0-ci/wet-boew/js/wet-boew.min.js:12:10356)
    at HTMLElement.<anonymous> (https://wet-boew.github.io/v4.0-ci/wet-boew/js/wet-boew.min.js:12:14261)
    at HTMLDocument.dispatch (https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js:3:7537)
    at HTMLDocument.r.handle (https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js:3:5620)
    at Object.trigger (https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js:4:4818)
    at HTMLElement.<anonymous> (https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js:4:5328)
    at Function.each (https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js:2:2861)
    at n.fn.init.each (https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js:2:845)
    at n.fn.init.trigger (https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js:4:5304)
```

The exception occurs with WET 4.0.39, and can be seen on [the English and French example pages](https://wet-boew.github.io/v4.0-ci/demos/tabs/tabs-en.html) in the documentation.

The tabbed interface still appears to render correctly, but the errror should not be occurring.

**Comment by ialexca** [comment]
I'm having the same error. Because of this problem, I had to rollback to version 4.0.38.

**Comment by thekodester** [comment]
Looks like https://github.com/wet-boew/wet-boew/pull/8948 broken this.

The issue is class name `tgl-panel` is used in too many plugins causing the src/plugins/toggle/toggle.js file to trigger.

https://github.com/wet-boew/wet-boew/blob/7f0186fee60171424ed46541928c2fc88d4d92d3/src/plugins/toggle/toggle.js#L18

**Comment by thekodester** [comment]
Thoughts on resolution? 

@piopi @duboisp @GormFrank

**Comment by piopi** [comment]
@thekodester , I see what is the problem. [Here](https://wet-boew.github.io/v4.0-ci/demos/toggle/toggle-en.html) the tablist is directly the parent of `<details>` , compared to this [ one ](https://wet-boew.github.io/v4.0-ci/demos/tabs/tabs-en.html) where tablist is not the parent of `<details>` . The scope of the wrapping was to only be applied to elements with the .accordion class. One possible solution is just to ensure that the parent of `<details>` elements has a class .accordion before doing any wrapping with a div. Adding more conditions to the if statement should solve. the issue https://github.com/wet-boew/wet-boew/blob/7f0186fee60171424ed46541928c2fc88d4d92d3/src/plugins/toggle/toggle.js#L123

For example: `if ( elm.nodeName.toLowerCase() === "details" &&  elm.parentNode.className.toLowerCase() === "accordion")`

**Comment by thekodester** [comment]
Good to know @piopi 

Also I think this brings up another issue as mentioned above with class name `tgl-panel`

It too commonly used, there quite a few functions that look for it in the toggle script that may trigger in other applications. I think some plugins use `tgl-panel` for the css only, maybe a different name should be added for css only use?

**Comment by piopi** [comment]
@thekodester I agree with you,  I could add a class for CSS only use but maybe the change should be handled in another ticket/issue, explaining why we need it.

**Comment by thekodester** [comment]
@piopi For sure should be two PRs!

**Comment by duboisp** [comment]
I might revert your change @piopi to fix the immediate functional issue and let you to submit a new PR with a more stable fix.

**Comment by piopi** [comment]
@duboisp I just pushed my changes all the issues are fixed.

**Comment by duboisp** [comment]
@ialexca it is now fixed in the version 4.0.39.1
